src/html/scripts/main.pug

```diff
@@ -23,12 +23,16 @@ script.
       onErr(err+" (MainRenderer:"+line+")");
   }
   // START
-
-  particles.show();
```

Comment:
It just makes startup a lot faster. This was actually not needed before, because the particles weren't hidden by default. No they are, made visible for the first time after the reboot to boot-loader, unless the user specifies otherwise. Some people (@Flohack74) complained that the particles used too much cpu, so i thought having the option to disable them would be nice.